build.sbt

```diff
@@ -104,7 +109,7 @@ lazy val dev = project
   .settings(
     name := "dev",
     libraryDependencies ++= Seq(
-      "com.github.scopt" %% "scopt" % "3.7.0"
+      "com.github.scopt" %% "scopt" % "4.0.1"
```

Comment:
Be worth double checking the CLI still works, because this is a major change. Off the top of my head one thing to make sure is that the help text still gets printed when things fail.

Historically `scopt` did a lot of these "automatically print helpful stuff for you", but doing this makes it really hard to test things. I gather the new version is trying to be a bit more explicit about how it should behave. It may be that you have to opt-in to the new behaviour and this still works as it did in `v3.x.y`, in which case we're all good!

Reply:
@adamnfish great tip!